jakttest/run_one.py

```diff
@@ -10,8 +10,8 @@
 from pathlib import Path
 
 # .lib on windows, .a otherwise
-MAIN_LIBRARY_NAME = "libjakt_main.a" if os.name != "nt" else "jakt_main.lib"
-RUNTIME_LIBRARY_NAME = "libjakt_runtime.a" if os.name != "nt" else "jakt_runtime.lib"
+MAIN_LIBRARY_NAME = "libjakt_main_%s.a" if os.name != "nt" else "jakt_main_%s.lib"
```

Comment:
wew, this is python3, we use f-strings or .format() instead of % strings in this household

Reply:
(if this is really the cleanest way then sure, but it strikes me as a python2-ism)

Reply:
okay, .format() is fine too